src/components/highlight/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { chakra, useToken } from '@chakra-ui/react';
+
+const Highlight: FC<PropsWithChildren> = ({ children }) => {
+  const [brandPrimary] = useToken('colors', ['brand.primary']);
+
+  return (
+    <chakra.span
```

Comment:
Is it possible to move the styles into the theme?
Yes -> it would be nice
No -> then you don't need the chakra factory tag but a normal html tag <span>. As fas as I remember